Dockerfile.test

```diff
@@ -0,0 +1,21 @@
+FROM python:3.6-stretch
+
+ARG DBOT_ARGS
+ARG ENVIRONMENT=production
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+RUN mkdir temp
+RUN mkdir shared
```

Comment:
Minor: `RUN mkdir temp && mkdir shared` or `RUN mkdir temp shared` will generate one less interim Docker image as they are a single atomic unit of work.